# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
# This example is compatible with Dynamixel using protocol 1.0

# port: /dev/u2d2
# u2d2_baud: 9600 # or 115200 or 1000000 i don't think this param is currently being used
# rate: 70
# read_rate: 70
# write_rate: 70

# TODO come back and check that these correspond to where the motors are located
#dynamixel_info:
left_drive_wheel_joint:
  ID: 1
  Return_Delay_Time: 0

  # Setting angle limits to zero activates 'wheel mode'
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 0

  Temperature_Limit: 80

right_drive_wheel_joint: 
  ID: 2
  Return_Delay_Time: 0

  # Setting angle limits to zero activates 'wheel mode'
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 0
  
  Temperature_Limit: 80